### PR TITLE
Docs: Enable users to disable preview for all when publishing Kedro-Viz

### PR DESCRIPTION
**Pull request by jitu5**
## Description

Docs added for disabled preview for all datasets under the Publish and share & Preview datasets.

Note: I have not updated the gif for publish popup, as some design changes is pending, I will update it later in this PR. 

## Checklist

- [X] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [X] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by astrojuanlu — APPROVED** [review]
💯

**Comment by astrojuanlu** [comment]
Rendered result:

https://kedro--1925.org.readthedocs.build/projects/kedro-viz/en/1925/platform_agnostic_sharing_with_kedro_viz.html

https://kedro--1925.org.readthedocs.build/projects/kedro-viz/en/1925/preview_datasets.html

File: docs/source/platform_agnostic_sharing_with_kedro_viz.md
Comment (rashidakanchwala) [inline docs/source/platform_agnostic_sharing_with_kedro_viz.md:44]: i think this might go in the next release 9.2.0

Comment (jitu5) [reply docs/source/platform_agnostic_sharing_with_kedro_viz.md:44]: Okay let me update to Kedro-Viz 9.2.0

Comment (jitu5) [reply docs/source/platform_agnostic_sharing_with_kedro_viz.md]: Done.

**Comment by astrojuanlu** [comment]
Anything missing here @jitu5 ? Or just approvals?

**Comment by jitu5** [comment]
> Anything missing here @jitu5 ? Or just approvals?

Yes, need to update gif for publish popup with disable preview toggle button.

File: docs/source/platform_agnostic_sharing_with_kedro_viz.md
Comment (ravi-kumar-pilla) [inline docs/source/platform_agnostic_sharing_with_kedro_viz.md:44]:
```suggestion
Starting from Kedro-Viz 9.2.0, `kedro viz build` will not include dataset previews by default. To enable previews for all the datasets, use the `--include-preview` flag.
```

File: docs/source/publish_and_share_kedro_viz_on_aws.md
Comment (ravi-kumar-pilla) [inline docs/source/publish_and_share_kedro_viz_on_aws.md:110]:
```suggestion
Starting from Kedro-Viz 9.2.0, `kedro viz deploy` will not include dataset previews by default. To enable previews for all the datasets, use the `--include-preview` flag.
```

File: docs/source/publish_and_share_kedro_viz_on_azure.md
Comment (ravi-kumar-pilla) [inline docs/source/publish_and_share_kedro_viz_on_azure.md:120]:
```suggestion
Starting from Kedro-Viz 9.2.0, `kedro viz deploy` will not include dataset previews by default. To enable previews for all the datasets, use the `--include-preview` flag.
```

File: docs/source/publish_and_share_kedro_viz_on_gcp.md
Comment (ravi-kumar-pilla) [inline docs/source/publish_and_share_kedro_viz_on_gcp.md:106]:
```suggestion
Starting from Kedro-Viz 9.2.0, `kedro viz deploy` will not include dataset previews by default. To enable previews for all the datasets, use the `--include-preview` flag.
```

**Comment by ravi-kumar-pilla** [comment]
Hi @jitu5 , I have left few suggestions on the docs. Apart from these, we also need to update the `README.md` file to have the new option `--include-previews` in the CLI commands usage [documentation](https://github.com/kedro-org/kedro-viz/blob/main/README.md#cli-usage). Thank you

**Comment by jitu5** [comment]
> Hi @jitu5 , I have left few suggestions on the docs. Apart from these, we also need to update the `README.md` file to have the new option `--include-preview` in the CLI commands usage [documentation](https://github.com/kedro-org/kedro-viz/blob/main/README.md#cli-usage). Thank you

@ravi-kumar-pilla Updated, Thanks.

File: README.md
Comment (ravi-kumar-pilla) [inline README.md:183]: Looking at this, I am thinking if we should make this flag a plural form (--include-previews ), similar to --include-hooks as this flag deals with all the dataset previews. Since we haven't shipped this yet, it would be good to address this now. What do you all think ? @jitu5 @stephkaiser @Huongg

File: README.md
Comment (ravi-kumar-pilla) [inline README.md:201]:
```suggestion
  --include-preview  A flag to include preview for all the datasets
```

File: README.md
Comment (jitu5) [reply README.md:183]: @ravi-kumar-pilla I think it make sense to make a plural form `--include-previews` as this flag deals with all the dataset previews.

Comment (ravi-kumar-pilla) [reply README.md:183]: should we make the change in - https://github.com/kedro-org/kedro-viz/pull/1895 ? @jitu5 once the team approves this change ?

Comment (jitu5) [reply README.md:183]: We can keep in https://github.com/kedro-org/kedro-viz/pull/1895

Comment (jitu5) [reply README.md:183]: @ravi-kumar-pilla I just had a quick chat with @stephkaiser and she is fine with a plural form `--include-previews` as flag.

Comment (ravi-kumar-pilla) [reply README.md:183]: Lets update the flag in #1895 and also the docs. thank you @jitu5

Comment (ravi-kumar-pilla) [reply README.md:183]: Let's hold on this. Sorry for the confusion. --include-preview seems to be for each dataset (since a dataset has only 1 preview, it seems more appropriate if we think that way) while --include-hooks was a project level thing.

**Review by ravi-kumar-pilla — APPROVED** [review]
LGTM. Thank you @jitu5

File: README.md
Comment (jitu5) [reply README.md]: @ravi-kumar-pilla As I discussed with @stephkaiser we agreed on `--include-previews` as flag. All the relevant changes in code I did in https://github.com/kedro-org/kedro-viz/pull/1895 PR and doc changes in this PR.

**Pull request merged**
